cmd/ncdu/ncdu.go

```diff
@@ -1026,11 +1017,16 @@ outer:
 				// Refresh the screen. Not obvious what key to map
 				// this onto, but ^L is a common choice.
 				case key(tcell.KeyCtrlL):
-					u.s.Sync()
+					goto redraw
 				}
 			}
 		}
-		// listen to key presses, etc.
+
+		u.Draw()
+		u.s.Show()
+	redraw:
+		u.Draw()
```

Comment:
Why does it `Draw `it twice?

Reply:
Oops. I forgot to do a `continue` before the label.

Maybe I shouldn't use a `goto`? I thought it would be more clear this way.

Reply:
I am an old C programmer so `goto exit` is very familiar.

However if it is easy to remove the goto then we probably should!